experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -1166,6 +1166,18 @@ def test_computed_end_date_returns_actual(self):
             experiment.end_date,
         )
 
+    def test_release_date_returns_as_expected(self):
+        release_date = datetime.date.today()
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.ENDING_APPROVE_APPROVE,
+            proposed_release_date=release_date,
+        )
+
+        self.assertEqual(
+            experiment.proposed_release_date,
+            release_date,
+        )
```

Comment:
Oh this doesn't actually test anything.  With this change you'd want to test against the API.

Reply:
I'm rebasing this right now this was supposed to be gone. I'll ping you when it's ready to be reviewed